### DISCUSSION TITLE
Can you help finish with-react-css-modules example ?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#17575] [Help] [answered]
Hi, i am trying to use babel-plugin-react-css-module
https://github.com/vercel/next.js/tree/1c20a9bfdd3fc913a42037d32f038ac83d2aad1e/examples/with-react-css-modules
I am trying to use it, it can generate hash class CSS but no CSS file load, so the style not work.  Thank you so much if you can help fix this error :( 
![image](https://user-images.githubusercontent.com/15695252/94962584-e55b7280-0520-11eb-90ac-071a7162888c.png)

**Comment by jamesmosier** [comment, marked as answer]
Next supports CSS modules by default: https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css

Install the latest version of Next to start using them with **no configuration or packages needed**.
